bin/ch/ch.vcxproj

```diff
@@ -27,6 +27,7 @@
         $(IntDir);
         %(AdditionalIncludeDirectories);
       </AdditionalIncludeDirectories>
+      <PreprocessToFile Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">false</PreprocessToFile>
```

Comment:
Yeah, good catch. This was left over when I was hunting down extended prefix exhaustion.